From 1d315aac4403f22274fceab3e22ec1cdb95bad78 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lawrence=20Vela=CC=81zquez?= <larryv@alum.mit.edu>
Date: Tue, 12 Mar 2013 03:34:14 -0400
Subject: [PATCH] Copy libsyntax from local Rust to stage0.

The local_stage0 script was not updated after commit 7dcbaed renamed
librustsyntax to libsyntax, so builds using local Rust fail due to
missing libsyntax.
---
 src/etc/local_stage0.sh |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/etc/local_stage0.sh b/src/etc/local_stage0.sh
index e4d7b61..5898bc5 100755
--- a/src/etc/local_stage0.sh
+++ b/src/etc/local_stage0.sh
@@ -46,3 +46,4 @@ cp ${PREFIX}/lib/rustc/${TARG_DIR}/${LIBDIR}/* ${TARG_DIR}/stage0/${LIBDIR}/
 cp ${PREFIX}/lib/librust*${LIB_SUF} ${TARG_DIR}/stage0/${LIBDIR}/
 cp ${PREFIX}/lib/libcore*${LIB_SUF} ${TARG_DIR}/stage0/${LIBDIR}/
 cp ${PREFIX}/lib/libstd*${LIB_SUF} ${TARG_DIR}/stage0/${LIBDIR}/
+cp ${PREFIX}/lib/libsyntax*${LIB_SUF} ${TARG_DIR}/stage0/${LIBDIR}/
-- 
1.7.10

